canonical-websites/cn.ubuntu.com
# Update ./run to the latest version

## Done
Updated `./run` to the latest django-db version.

## QA
- Check that this branch creates a demo
- Run `./run`
- Run `./run clean`
- Run `./run test`
- Check that all work as expected

## Issue / Card
Fixes https://github.com/ubuntudesign/vanilla-squad/issues/76
